internal/services/containers/container_group_resource.go

```diff
@@ -123,6 +123,29 @@ func resourceContainerGroup() *pluginsdk.Resource {
 
 			"identity": commonschema.SystemAssignedUserAssignedIdentityOptional(),
 
+			"network_profile_id": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: networkValidate.NetworkProfileID,
+				Deprecated:   "the 'network_profile_id' has been deprecated and will be removed from the 4.0 AzureRM provider. Please use the 'subnet_id' field instead",
```

Comment:
shall we call out this no longer does anything?
```suggestion
				Deprecated:   "the 'network_profile_id' has been remove from the latest versions of the container server API and has been deprecated. It no longer functions  and will be removed from the 4.0 AzureRM provider. Please use the 'subnet_id' field instead",
```

Reply:
@katbyte Added the note to deprecation message.